client/pkg/client/openapi/api_workflow_runs.go

```diff
@@ -12,74 +12,67 @@ package openapi
 
 import (
 	"bytes"
-	_context "context"
-	_ioutil "io/ioutil"
-	_nethttp "net/http"
-	_neturl "net/url"
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
 	"os"
 	"strings"
 )
 
-// Linger please
-var (
-	_ _context.Context
-)
-
 // WorkflowRunsApiService WorkflowRunsApi service
 type WorkflowRunsApiService service
 
-type ApiGetWorkflowRunRequest struct {
-	ctx               _context.Context
+type WorkflowRunsApiGetWorkflowRunRequest struct {
+	ctx               context.Context
 	ApiService        *WorkflowRunsApiService
-	workflowName      string
+	workflowPath      GetWorkflowSubscriptionsWorkflowPathParameter
```

Comment:
When the `workflowPath.String` (which is a `*String`) is printed by `parameterToString` with `fmt.Sprintf("%v"...` it ends up returning the address rather than the value. I think this is another bug in openapi-generator. It seems to happen if a path parameter is a `OneOf`.

Reply:
I changed our openapi spec to work around this... just not using `OneOf` on path parameters for now.